woocommerce/woocommerce-ios
# Dynamic Dashboard: Update Blaze card UI

Closes: #12488


## Description


This PR updates the UI for Blaze card on dashboard to match the Figma design:
<img width="445" alt="Screenshot 2024-04-18 at 10 59 40" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/03d2c327-3dde-49ba-863a-d197fc78bd28">

## Testing instructions

1. Run app,
2. ensure Blaze card looks like screenshot below,
3. test and ensure all functionality still works (except for Hide Blaze, still TODO)

| No campaign | Existing campaign |
|-|-|
| <img src="https://github.com/woocommerce/woocommerce-ios/assets/266376/2eb46669-ebe6-4721-b6ae-3a8a7106794b"> | <img src="https://github.com/woocommerce/woocommerce-ios/assets/266376/eb2a617c-0567-4639-96fd-8b7abb8d7cde"> |
